typescript/infra/config/environments/mainnet3/owners.ts

```diff
@@ -55,6 +55,7 @@ export const owners: ChainMap<OwnableConfig> = Object.fromEntries(
         proxyAdmin: timelocks[local] ?? safes[local] ?? DEPLOYER,
         validatorAnnounce: DEPLOYER, // unused
         testRecipient: DEPLOYER,
+        fallbackRoutingHook: DEPLOYER,
```

Comment:
is this so we dont transfer to the safe?

Reply:
yeah otherwise we can't really use the current tooling to fully connect, bc we don't have hook governor implemented

I'm not too sure why when we did the ownership transfer earlier we didn't end up transferring ownership of these hooks to the safes tbh - but just adding it to be extra sure